packages/rpc-methods/src/permitted/common/snapInstallation.ts

```diff
@@ -2,7 +2,9 @@ import { ethErrors } from 'eth-rpc-errors';
 import { RequestedPermissions } from '@metamask/controllers';
 import { InstallSnapsResult } from '@metamask/snap-controllers';
 import { isObject } from '@metamask/utils';
-import { SNAP_PREFIX } from '@metamask/snap-utils';
+
+// TODO: Remove
+const SNAP_PREFIX = 'wallet_snap_';
```

Comment:
I added these to test the `rpc-methods` package in the extension. The current version of `snap-utils` used in the extension doesn't have the `SNAP_PREFIX`.

Reply:
Ah, is the idea that we'll bump `snap-utils` in the extension? We just need to make sure to remove all instances of this before merging.

Reply:
Yeah, I added the TODOs as a reminder for myself.